docs/kubernetes/README.md

```diff
@@ -0,0 +1,27 @@
+# Kontain Integration Into Kubernetes
+
+* See [Quick Start Guide](quickstart.md) for a fast demonstration of Kontain integrated with Kubernetes.
+
+Kubernetes has a mechanism called Runtime Classes that allows new container types to be introduced. Kontain uses
+a Kubernetes Runtime Class to introduce itself into the kubernetes ecology. Once introduced, users specify Kontain
+as the runtime environment for their container(s) by adding `runtimeClassName: kontain` to their YAML Pod specifications.
+
+Kontain requires that the Kubernetes host nodes run an OCI compliant container manager like CRI-O or `containerd`.
+When installed, Kontain modifies the host container manager to recognize start requests for the meant for the Kontain
```

Comment:
Missing word or extra words: "to recognize start requests for the (meant for the?) Kontain

Reply:
done